packages/manager/modules/vrack/src/vrack.controller.js

```diff
@@ -249,10 +263,48 @@ export default class VrackMoveDialogCtrl {
           return services;
         });
 
+        if (has(allServices, 'dedicatedCloud')) {
+          const groupedDedicatedCloud = groupBy(
+            allServices.dedicatedCloud,
+            (dedicatedCloud) => {
+              return dedicatedCloud.productReference === 'MBM'
+                ? 'managedBareMetal'
+                : 'dedicatedCloud';
+            },
+          );
+          allServices = assign(
+            allServices,
+            { dedicatedCloud: [] },
+            groupedDedicatedCloud,
+          );
```

Comment:
```suggestion
          allServices = {
            ...allServices,
            { dedicatedCloud: [] },
            ...groupedDedicatedCloud,
          };
```
or maybe use `Object.assign` ?

Reply:
Thanks for the suggestion.
Maybe it could be reviewed on a separate Pull Request regarding how Lodash is
used and can be tackled once.

For information:
```sh
$ git grep 'lodash/assign' -- '*.js' | wc -l
111
```